app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,84 @@
-.check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
+
+.govuk-check-your-answers {
+
+  > * {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
+
+    > * {
+      display: block;
+    }
   }
 
-  .change-answer {
-    text-align: right;
+  .cya-question {
     font-weight: bold;
```

Comment:
No, I think our font mixins are used too often, which makes things less flexible and increases the CSS file size for no good reason.
If we had a variable for bold, that would be worth using, though. I couldn't find it, did I only dream about it?